scripts/install/installRedis.sh

```diff
@@ -4,4 +4,4 @@ wget http://download.redis.io/redis-stable.tar.gz
 tar xvzf redis-stable.tar.gz
 cd redis-stable
 sudo make install
-src/redis-server > /dev/null &
+src/redis-server > /dev/null &
```

Comment:
@paulodamaso Is this line redirecting the output to ``/dev/null``? If yes, why? I guess we need to see any possible logs somehow.